Podcast/UserEpisodeData.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  UserEpisodeData.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 5/9/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+// essentially tuples to store our hashmap of blurbs (swift tuples aren't hashable)
+struct EpisodeToUser: Hashable {
+    let episodeID: String
+    let userID: String
+
+    var hashValue: Int {
+        return "\(episodeID), \(userID)".hashValue
+    }
+
+    static func == (lhs: EpisodeToUser, rhs: EpisodeToUser) -> Bool {
+        return lhs.episodeID == rhs.episodeID && lhs.userID == rhs.userID
+    }
+}
+
+// all info about users to episodes
+class UserEpisodeData: NSObject {
+
+    static var shared: UserEpisodeData = UserEpisodeData()
+
+    var blurbs: [EpisodeToUser: String]
```

Comment:
What about just string to string and have the key be episodeId + userId like you did? Have the access functions pass in both the user id and episode id, we can do whatever we want internally.